.betterer.results

```diff
@@ -938,6 +938,12 @@ exports[`better eslint`] = {
     "packages/grafana-ui/src/components/VizLegend/VizLegendListItem.tsx:5381": [
       [0, 0, 0, "Use data-testid for E2E selectors instead of aria-label", "0"]
     ],
+    "packages/grafana-ui/src/components/VizLegend/VizLegendTable.tsx:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"],
+      [0, 0, 0, "Do not use any type assertions.", "1"],
+      [0, 0, 0, "Do not use any type assertions.", "2"],
+      [0, 0, 0, "Do not use any type assertions.", "3"]
```

Comment:
i'm not sure how to TS my way out of this without adding unnecessary coercion code :cry: